lib/send.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+// utility to send messages to processes
+function send(ps, name, data) {
+	if (typeof ps === 'string') {
```

Comment:
Is this really needed? Do we really pass a custom `process` anywhere? And even if we do, wouldn't it be better to just have it as the third argument instead? For simplicity.